tests/foreman/cli/test_host.py

```diff
@@ -161,17 +162,15 @@ def test_positive_create_with_name(self):
             with self.subTest(name):
                 host = entities.Host()
                 host.create_missing()
-                result = Host.create({
+                result = make_host({
                     u'architecture-id': host.architecture.id,
                     u'domain-id': host.domain.id,
                     u'environment-id': host.environment.id,
-                    # pylint:disable=no-member
```

Comment:
Okay, I had no particular logic in my thoughts =), but according debug I do not see any difference between `host.organization`, `host.domain` or `host.medium`. Is it still true statement for `organization` and `location`?